conda/environments/cudf_dev_cuda10.yml

```
@@ -0,0 +1,29 @@
+name: cudf_dev
+channels:
+- nvidia/label/cuda10.0
+- rapidsai/label/cuda10.0
+- numba
+- conda-forge
+- defaults
+dependencies:
+- cmake>=3.12
+- python>=3.6,<3.8
+- numba>=0.40
+- pandas>=0.23.4
+- pyarrow=0.12.1
+- notebook>=0.5.0
+- boost
+- nvstrings>=0.3.0
```

I think we should ping this to prevent pulling a newer than compatible NVStrings version:
```suggestion
- nvstrings>=0.3,<0.4
```